x-pack/legacy/plugins/canvas/canvas_plugin_src/strings/functions/gte.ts

```diff
@@ -11,11 +11,17 @@ import { FunctionFactory } from '../../functions/types';
 
 export const help: FunctionHelp<FunctionFactory<typeof gte>> = {
   help: i18n.translate('xpack.canvas.functions.gteHelpText', {
-    defaultMessage: 'Return if the context is greater than or equal to the argument',
+    defaultMessage: 'Returns whether the {context} is greater or equal to the argument.',
+    values: {
+      context: '_context_',
```

Comment:
nit: are these underscores converted with markdown somehow?  Or should there be code tick marks in the default message instead?

Reply:
Currently autocomplete in Canvas doesn't support markdown, but I filed an issue a while back to add support for markdown rendering. https://github.com/elastic/kibana/issues/38439

This is the same syntax used in asciidoc, so it will show up as italicized in the published docs. For the sake of keeping all the content in sync including text formatting, I kept the syntax in even though you see the raw characters in the autocomplete.